packages/ui/components/form/timezone-select/TimezoneSelect.tsx

```diff
@@ -21,6 +21,12 @@ export function TimezoneSelect({ className, components, ...props }: SelectProps)
         "America/Asuncion": "Asuncion",
       }}
       {...props}
+      formatOptionLabel={(option) => <p className="truncate">{(option as ITimezoneOption).value}</p>}
+      getOptionLabel={(data) => {
+        const option = data as ITimezoneOption;
+        const formatedLabel = option.label.split(")")[0].replace("(", " ").replace("T", "T ");
+        return `${option.value}${formatedLabel}`;
+      }}
```

Comment:
Hi @emrysal, the e2e test is failing because of the updated label format in `TimezoneSelect` that can be fixed by updating the label text in the test file. But before pushing the fix following needs confirmation:
Should the updated label format only be limited to the `AvailabilityPage` or should it be consistent throughout the app (as currently)? cc: @PeerRich @Jaibles

Reply:
@emrysal can help fix the test. i rather not have externals touch tests for now